ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -95,7 +95,8 @@ type RayJobStatus struct {
 	// It is not guaranteed to be set in happens-before order across separate operations.
 	// It is represented in RFC3339 form
 	StartTime *metav1.Time `json:"startTime,omitempty"`
-	// Represents time when the job was ended.
+	// EndTime is the time when status.jobDeploymentStatus transitioned to 'Complete' status.
+	// This occurs when the Ray job reaches a terminal state (SUCCEEDED, FAILED, STOPPED)
```

Comment:
There are two possibilities to transition to Complete at this moment. See https://github.com/ray-project/kuberay/pull/1872#pullrequestreview-1845027314 for more details.

Reply:
> There are two possibilities to transition to Complete at this moment

Right, but both possibilities still result in the Ray job reaching a terminal state right? Unless I'm missing something

Reply:
Ah, I see what you mean. The second condition when submitter job fails, is irrelevant to the Ray job status

Reply:
I updated the comments to also include when submitter job fails